features/step_definitions/link_pension_webchat_to_pension_and_retirement_pages_steps.rb

```diff
@@ -1,18 +1,39 @@
-Given(/^I visit an article( not| NOT)? within the Pension and Retirement category in my "([^"]*)"$/) do |should_not, language|
-	article =
-    if should_not
-      {
-        'English' => '/en/articles/saving-money-for-christmas',
-        'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
-      }
-    else
-      {
-        'English' => '/en/articles/personal-pensions',
-        'Welsh' => '/cy/articles/pensiynau-personol'
-      }
-    end
+Given(/^I visit "([^"]*)" not within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
+  links = {
+    'article' => {
+      'English' => '/en/articles/saving-money-for-christmas',
+      'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
+    },
+    'category' => {
+      'English' => '/en/categories/help-with-your-benefits',
+      'Welsh' => '/cy/categories/help-with-your-benefits'
+    },
+    'tool' => {
+      'English' => '/en/tools/budget-planner',
+      'Welsh' => '/cy/tools/cynllunydd-cyllideb/start'
+    }
+  }
 
-  visit(article[language])
+  visit(links[entity][language])
+end
+
+Given(/^I visit "([^"]*)" within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
```

Comment:
@gotbadger some comment as above. We can also move this hash into a private method into a private method. This is just a suggestion. It's not a deal breaker, just a nice to have.